src/app/api/services/dot-languages/dot-languages.service.ts

```diff
@@ -0,0 +1,26 @@
+import { Injectable } from '@angular/core';
+import { Observable } from 'rxjs/Observable';
+import { Language } from '../../../shared/models/language/language.model';
+
+/**
+ * Provide util methods to get Languages available in the system.
+ * @export
+ * @class DotLanguagesService
+ */
+@Injectable()
+export class DotLanguagesService {
+    constructor() {}
+
+    /**
+     * Return languages.
+     * @returns {Observable<Language[]>}
+     * @memberof DotLanguagesService
+     */
+    get(): Observable<Language[]> {
+        return Observable.of([
+            { id: 'en', label: 'English' },
+            { id: 'es', label: 'Spanish' },
+            { id: 'fr', label: 'French' }
+        ]);
```

Comment:
this have to be request from the BE in the future, so add A TODO here

Reply:
Yeah we have this on the BE.
```
demo.dotcms.com/api/v1/languages

{
  "en": {
    "name": "English"
  },
  "es": {
    "name": "Espanol"
  }
}
```